test/DefaultInvokerTest.php

```diff
@@ -62,7 +78,34 @@ public function setup() {
 
 	}
 
+	/** @expectedException Fliglio\Routing\RouteException */
+	public function testConnectingDuplicateRoutes() {
+		// when
+		$this->routeMap->connect('postEx', 
+			RouteBuilder::get()
+				->uri('/foo')
+				->command('Fliglio\Routing.StubResource.addFoo')
+				->method(Http::METHOD_POST)
+				->build()
+		);
+	}
+
+	/** @expectedException Fliglio\Flfc\Exceptions\RedirectException */
+	public function testChangeOfProtocol() {
+		// given
+		$this->request->setUrl('/biz/123');
```

Comment:
Thanks for adding test. Add request with GET params?

Reply:
Added a second test. One with and one without.